src/csharp/Grpc.Core/Internal/NativeExtension.cs

```diff
@@ -106,7 +106,9 @@ private static UnmanagedLibrary LoadUnmanagedLibrary()
         /// </summary>
         private static NativeMethods LoadNativeMethods()
         {
-            return PlatformApis.IsUnity ? LoadNativeMethodsUnity() : new NativeMethods(LoadUnmanagedLibrary());
+            return PlatformApis.IsUnity ? LoadNativeMethodsUnity() :
```

Comment:
nit: maybe just use an if statement instead of one of the `?` operators?  Nested ternary operator seems hard to read.

Reply:
Fixed with commit be6f3e393fdbf80f01b3.